clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -644,20 +649,26 @@ trait GvcfTests {
     ).map {
       case (path, contents) => Files.write(path, contents.getBytes)
     }
+
+    if (testNonExistingFile) IoUtil.deleteGoogleObject(gvcfPath.toUri)
```

Comment:
Seems more straightforward to not `Files.write` above at all if `testNonExistingFile` is true, rather than writing then deleting.

Reply:
Yeah I think the only reason I didn't do this is because it write metrics files as well which it then checks later to make sure the are around. It might make more sense to split this off from testDeleteGvcf entirely

Reply:
On the other hand, this tests the actual scenario that led to the problem.